### PR TITLE
Add sidebar icons

### PR DESCRIPTION
**Pull request by KBoohene**
Added icons to project using material ui icons

### CODE REVIEW

File: src/HomePage.js [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
-import WelcomePage from './WelcomePage';
-import AboutPage from './AboutPage';
+import React from "react";
+import { Route, withRouter } from "react-router-dom";
+import styled from "styled-components";
+import Sidebar from "./Sidebar";
+import WelcomePage from "./WelcomePage";
+import AboutPage from "./AboutPage";
+import HomeIcon from "@material-ui/icons/Home";
+import InfoIcon from "@material-ui/icons/Info";
```
Comment (fosco) [inline src/HomePage.js]: It would be better to have single quotes in the imports

File: src/Sidebar.js [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-  height:40px;
-  width:100%;
+  height: 40px;
+  width: 100%;
   padding-left: 20px;
   padding-right: 20px;
   text-decoration: none;
   color: black;
-  ${(props) => props.selected && `background-color: ${DARK_GRAY};`}
+  ${props => props.selected && `background-color: ${DARK_GRAY};`};
```
Comment (fosco) [inline src/Sidebar.js]: The semicolon at the end is not needed. Also, the parentheses around `props` are a good habit and should be kept

File: src/Sidebar.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ const Label = styled.span`
 `;
 
 const context = createContext({
-  currentPath: '/',
+  currentPath: "/"
```
Comment (fosco) [inline src/Sidebar.js]: Trailing commas are a good habit, they should be kept

**Comment by riourbana** [comment]
Thank you for the PR @KBoohene, could you fix all the linting stuff?

**Review by fosco — CHANGES_REQUESTED** [review]
Thank you for the PR @KBoohene, could you fix all the linting stuff?

File: src/Sidebar.js (hunk 3 above)
Comment (KBoohene) [reply src/Sidebar.js]: Sorry about that, it looks like its a configuration from my editor

**Review by KBoohene — COMMENTED** [review]
I have completed the linting changes, sorry for the issue

**Comment by fosco** [comment]
Merged. Nice job, thanks!

**Pull request merged**